input/examples/practitioner-bobrester-bob.xml

```diff
@@ -7,25 +7,16 @@
     <identifier>
         <type>
             <coding>
-                <system value="http://terminology.hl7.org.au/CodeSystem/v2-0203"/>
-                <code value="EI"/>
+                <system value="http://terminology.hl7.org/CodeSystem/v2-0203"/>
+                <code value="EN"/>
```

Comment:
Change code back to EI as per [AU Employee Number](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-employeenumber.html)

Reply:
done